src/test/java/de/bonndan/nivio/input/IndexerIntegrationTest.java

```diff
@@ -30,7 +29,7 @@
 @SpringBootTest
 @ExtendWith(MockitoExtension.class)
 @ActiveProfiles("test")
-@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+@TestMethodOrder(Alphanumeric.class)
```

Comment:
Thanks for your efforts, but method ordering is not necessary anymore. I've checked that the tests run without it. It also does not make sense anymore because nivio works differently now since when method ordering was introduced.

I think you can remove it.

Reply:
Good, point, will do.

Reply:
Done.
Leaving the conversation unresolved for your review.